### PR TITLE
Daniel --> master

### PR DESCRIPTION
**Pull request by YoungMasterGandalf**
This PR introduces 2 hotfixes:
* `Table` class init
* `AbstractTable` class init

Both needed for proper functionality (fix crashes) - instantiating a DB in FL, DB nodes functionality etc. However a deeper thought about the implementation is probably needed (they are just hotfixes after all).

### CODE REVIEW

File: dbhydra/src/abstract_table.py [hunk 1]
```diff
@@ -239,7 +239,12 @@ def __init__(self, db1, name, columns = None, types: Optional[list[str]] = None,
         self.types: list[str] = types #[] if types is None else types - is wrong, if types not initialized it should be None, not empty list
         self.id_column_name: str = id_column_name
         if self.columns is not None and self.types is not None:
-            assert len(self.columns) == len(self.types) 
+            assert len(self.columns) == len(self.types)
+            
+            # Hotfix: flatten nested lists (otherwise crashes might happen)
+            # TODO: Search for the causes and implement better handling
+            for i, column in enumerate(self.columns):
+                self.columns[i] = column[0] if type(column) == list else column
```
Comment (DovaX) [inline dbhydra/src/abstract_table.py:247]: I dont like this, why should the information be truncated? When can this even happen that column from columns would contain a list inside?

File: dbhydra/src/tables.py [hunk 2]
```diff
@@ -591,9 +591,14 @@ def get_foreign_keys_for_table(self, table_dict, foreign_keys):
 
 class Table(SqlServerTable):
     """Deprecated - do not remove until dbhydra 3.x"""
-    def __init__(self, config_file="config.ini", db_details=None):
-        print("Deprecation warning!, Table was renamed to SqlServerTable and the old name will deprecated in future!")
-        super().__init__(config_file=config_file, db_details=db_details)
+    def __init__(self, db1, name, columns=None, types=None):
+        super().__init__(db1, name, columns)
+        self.types = types
```
Comment (DovaX) [inline dbhydra/src/tables.py:596]: Deprecation warning shouldnt be removed

Comment (DovaX) [reply dbhydra/src/tables.py:596]: shouldnt super.__init__ include also "types"?

**Review by DovaX — CHANGES_REQUESTED** [review]

**Comment by DovaX** [comment]
I merged it but it shouldnt remain in the code for long time @YoungMasterGandalf

**Comment by DovaX** [comment]
Please prepare fix PR

**Comment by YoungMasterGandalf** [comment]
@DovaX Yes, I will it ASAP

**Pull request merged**